### PR TITLE
Clarify wording in README and cleanup unused code

### PR DESCRIPTION
**Pull request by runcom**
**- What I did**

reworded a sentence in the README and removed some really not used code.

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ both [the installer](https://github.com/openshift/installer/) as well as Red Hat
 CoreOS. See also [the machine-api-operator](https://github.com/openshift/machine-api-operator)
 which handles provisioning of new machines - once the machine-api-operator
 provisions a machine (with a "pristine" base Red hat CoreOS), the MCO will take
-care of configuring it.
+care of serving the configuration.
```
Comment (runcom) [inline README.md]: Changed this cause it's really misleading to think that we play an active role in the very first configuration of a new node. We just serve the generated machineconfigs via the mcs (but those were generated already on our side, bootstrap or running cluster). There's no active role we're playing here.

Comment (cgwalters) [reply README.md]: The base configuration that turns into the Ignition that configures the system does live in this operator though...

File: pkg/operator/bootstrap.go [hunk 2]
```diff
@@ -21,9 +21,10 @@ import (
 
 // RenderBootstrap writes to destinationDir static Pods.
 func RenderBootstrap(
-	clusterConfigConfigMapFile string,
-	infraFile, networkFile string,
-	etcdCAFile, rootCAFile string, kubeCAFile string, pullSecretFile string,
+	clusterConfigConfigMapFile,
+	infraFile, networkFile,
+	etcdCAFile, rootCAFile,
+	kubeCAFile, pullSecretFile string,
```
Comment (ashcrow) [inline pkg/operator/bootstrap.go]: :pray:

File: README.md (hunk 1 above)
Comment (runcom) [reply README.md]: yea but the sentence says that "provision a machine and MCO will take care of configuring" which is not the case... the operator bootstrap for instance, already took care of rendering the ignitions and serving them through the mcs by the time the machines provision

Comment (runcom) [reply README.md]: what you say is also true of course, but if we want to explain that we need to further fix the whole paragraph

Comment (cgwalters) [reply README.md]: I guess when I write "MCO" I'm usually referring to "all of the components" including the MCS.  

The other thing here is that not only do we provide the Ignition on boot, we take care of configuration *after* it's booted the first time too.  The "configuring it" here covers both those cases.

Comment (cgwalters) [reply README.md]: (BTW if you revise this PR can you also `s/Red hat/Red Hat/` above this?)

Comment (runcom) [reply README.md]: > The other thing here is that not only do we provide the Ignition on boot, we take care of configuration _after_ it's booted the first time too. The "configuring it" here covers both those cases.

yes, the thing is, explaining that with just "The MCO takes care of configuring it" and talking about machine-api-operator in the same sentence is just too simplistic and have already created confusion on how all this works (had a lengthy discussion yesterday about this sentence,that's why I wanted to clarify but I guess that requires a full rewrite of this paragraph)
I'm reverting this for now, corrected RH typo instead

**Comment by ashcrow** [comment]
Bootstrap failures

/retest

**Comment by kikisdeliveryservice** [comment]
bootstrap, trying again

/retest

**Comment by kikisdeliveryservice** [comment]
known flake: HAProxy
/test e2e-aws

**Comment by kikisdeliveryservice** [comment]
Throttling, going to hold off..

**Comment by ashcrow** [comment]
Rebase needed.

**Comment by runcom** [comment]
> Rebase needed.

:thinking:

**Comment by ashcrow** [comment]
Weird, it was showing a merge conflict in the UI but it's gone now :confused:

**Comment by kikisdeliveryservice** [comment]
i believe these errors are rate limiting, we're hitting it in other prs, holding off on retesting for now, will try again later.

**Comment by runcom** [comment]
This is finally all green

**Comment by LorbusChris** [comment]
lgtm, merge?

**Comment by kikisdeliveryservice** [comment]
In an effort to clean up the MCO repo, closing old open PRs with no recent activity.  

Feel free to reopen.

**Pull request closed without merging**